### ISSUE TITLE
[Starting a project steps] I cannot select an issuer in the "Do you know you're great?" page

### ISSUE THREAD

**Issue by DavidBruant** [Agoric/agoric-sdk#972]
In the VM shell, i can see a bunch of errors:
```sh
web: 127.0.0.1:59060[2]: inbound error: No handler for /private/wallet-bridge undefined from {
  "topic": "mg-installEventListeners",
  "data": {},
  "dappOrigin": "http://127.0.0.1:3000",
  "meta": {
    "channelID": 2,
    "origin": "http://localhost:8000",
    "url": "/private/wallet-bridge",
    "date": 1587394064156
  }
}
web: 127.0.0.1:59060[2]: inbound error: No handler for /private/wallet-bridge undefined from {
  "topic": "mg-frameInfo",
  "data": {
    "frameScrolling": null
  },
  "dappOrigin": "http://127.0.0.1:3000",
  "meta": {
    "channelID": 2,
    "origin": "http://localhost:8000",
    "url": "/private/wallet-bridge",
    "date": 1587394064157
  }
}
web: 127.0.0.1:59060[2]: inbound error: No handler for /private/wallet-bridge undefined from {
  "topic": "mg-installEventListeners",
  "data": {},
  "dappOrigin": "http://127.0.0.1:3000",
  "meta": {
    "channelID": 2,
    "origin": "http://localhost:8000",
    "url": "/private/wallet-bridge",
    "date": 1587394064157
  }
}
web: 127.0.0.1:59060[2]: inbound error: No handler for /private/wallet-bridge undefined from {
  "topic": "mg-frameInfo",
  "data": {
    "frameScrolling": null
  },
  "dappOrigin": "http://127.0.0.1:3000",
  "meta": {
    "channelID": 2,
    "origin": "http://localhost:8000",
    "url": "/private/wallet-bridge",
    "date": 1587394064157
  }
}
web: 127.0.0.1:59060[2]: inbound error: No handler for /private/wallet-bridge undefined from {
  "topic": "mg-installEventListeners",
  "data": {},
  "dappOrigin": "http://127.0.0.1:3000",
  "meta": {
    "channelID": 2,
    "origin": "http://localhost:8000",
    "url": "/private/wallet-bridge",
    "date": 1587394064157
  }
}
web: 127.0.0.1:59060[2]: inbound error: target[onError] does not exist, has onOpen,onClose,onMessage from {
  "topic": "mg-installEventListeners",
  "data": {},
  "dappOrigin": "http://127.0.0.1:3000",
  "error": "\"No handler for /private/wallet-bridge undefined\"",
  "meta": {
    "channelID": 2,
    "dispatcher": "onError",
    "origin": "http://localhost:8000",
    "url": "/private/wallet-bridge",
    "date": 1587394064339
  }
}
web: 127.0.0.1:59060[2]: inbound error: target[onError] does not exist, has onOpen,onClose,onMessage from {
  "topic": "mg-frameInfo",
  "data": {
    "frameScrolling": null
  },
  "dappOrigin": "http://127.0.0.1:3000",
  "error": "\"No handler for /private/wallet-bridge undefined\"",
  "meta": {
    "channelID": 2,
    "dispatcher": "onError",
    "origin": "http://localhost:8000",
    "url": "/private/wallet-bridge",
    "date": 1587394064419
  }
}
web: 127.0.0.1:59060[2]: inbound error: target[onError] does not exist, has onOpen,onClose,onMessage from {
  "topic": "mg-installEventListeners",
  "data": {},
  "dappOrigin": "http://127.0.0.1:3000",
  "error": "\"No handler for /private/wallet-bridge undefined\"",
  "meta": {
    "channelID": 2,
    "dispatcher": "onError",
    "origin": "http://localhost:8000",
    "url": "/private/wallet-bridge",
    "date": 1587394064513
  }
}
web: 127.0.0.1:59060[2]: inbound error: target[onError] does not exist, has onOpen,onClose,onMessage from {
  "topic": "mg-frameInfo",
  "data": {
    "frameScrolling": null
  },
  "dappOrigin": "http://127.0.0.1:3000",
  "error": "\"No handler for /private/wallet-bridge undefined\"",
  "meta": {
    "channelID": 2,
    "dispatcher": "onError",
    "origin": "http://localhost:8000",
    "url": "/private/wallet-bridge",
    "date": 1587394064590
  }
}
web: 127.0.0.1:59060[2]: inbound error: target[onError] does not exist, has onOpen,onClose,onMessage from {
  "topic": "mg-installEventListeners",
  "data": {},
  "dappOrigin": "http://127.0.0.1:3000",
  "error": "\"No handler for /private/wallet-bridge undefined\"",
  "meta": {
    "channelID": 2,
    "dispatcher": "onError",
    "origin": "http://localhost:8000",
    "url": "/private/wallet-bridge",
    "date": 1587394064666
  }
}
SwingSet: vat-wallet: Adding adminHandle Object <[Object: null prototype] {}> {
  dispatcher: 'onOpen',
  origin: 'http://127.0.0.1:8000',
  url: '/private/wallet',
  date: 1587394101679,
  channelHandle: [Object: null prototype] {
    toString: [Function (anonymous)] Function <Function <Complex prototype>>
  }
}
web: 127.0.0.1:59138[6]: inbound error: No handler for /private/wallet-bridge undefined from {
  "topic": "mg-installEventListeners",
  "data": {},
  "dappOrigin": "http://127.0.0.1:3000",
  "meta": {
    "channelID": 6,
    "origin": "http://localhost:8000",
    "url": "/private/wallet-bridge",
    "date": 1587394276248
  }
}
web: 127.0.0.1:59138[6]: inbound error: No handler for /private/wallet-bridge undefined from {
  "topic": "mg-frameInfo",
  "data": {
    "frameScrolling": null
  },
  "dappOrigin": "http://127.0.0.1:3000",
  "meta": {
    "channelID": 6,
    "origin": "http://localhost:8000",
    "url": "/private/wallet-bridge",
    "date": 1587394276248
  }
}
web: 127.0.0.1:59138[6]: inbound error: No handler for /private/wallet-bridge undefined from {
  "topic": "mg-installEventListeners",
  "data": {},
  "dappOrigin": "http://127.0.0.1:3000",
  "meta": {
    "channelID": 6,
    "origin": "http://localhost:8000",
    "url": "/private/wallet-bridge",
    "date": 1587394276249
  }
}
web: 127.0.0.1:59138[6]: inbound error: target[onError] does not exist, has onOpen,onClose,onMessage from {
  "topic": "mg-installEventListeners",
  "data": {},
  "dappOrigin": "http://127.0.0.1:3000",
  "error": "\"No handler for /private/wallet-bridge undefined\"",
  "meta": {
    "channelID": 6,
    "dispatcher": "onError",
    "origin": "http://localhost:8000",
    "url": "/private/wallet-bridge",
    "date": 1587394276473
  }
}
web: 127.0.0.1:59138[6]: inbound error: target[onError] does not exist, has onOpen,onClose,onMessage from {
  "topic": "mg-frameInfo",
  "data": {
    "frameScrolling": null
  },
  "dappOrigin": "http://127.0.0.1:3000",
  "error": "\"No handler for /private/wallet-bridge undefined\"",
  "meta": {
    "channelID": 6,
    "dispatcher": "onError",
    "origin": "http://localhost:8000",
    "url": "/private/wallet-bridge",
    "date": 1587394276650
  }
}
web: 127.0.0.1:59138[6]: inbound error: target[onError] does not exist, has onOpen,onClose,onMessage from {
  "topic": "mg-installEventListeners",
  "data": {},
  "dappOrigin": "http://127.0.0.1:3000",
  "error": "\"No handler for /private/wallet-bridge undefined\"",
  "meta": {
    "channelID": 6,
    "dispatcher": "onError",
    "origin": "http://localhost:8000",
    "url": "/private/wallet-bridge",
    "date": 1587394276736
  }
}
SwingSet: vat-wallet: Removing adminHandle Object <[Object: null prototype] {}> {
  dispatcher: 'onClose',
  origin: 'http://127.0.0.1:8000',
  url: '/private/wallet',
  date: 1587394279915,
  channelHandle: [Object: null prototype] {
    toString: [Function (anonymous)] Function <Function <Complex prototype>>
  }
}
SwingSet: vat-wallet: Adding adminHandle Object <[Object: null prototype] {}> {
  dispatcher: 'onOpen',
  origin: 'http://127.0.0.1:8000',
  url: '/private/wallet',
  date: 1587394280750,
  channelHandle: [Object: null prototype] {
    toString: [Function (anonymous)] Function <Function <Complex prototype>>
  }
}

```

**Comment by katelynsills** [comment]
Hey @DavidBruant, I just ran through the instructions and I'm not seeing these errors. Can we pair on this to debug what you're seeing?
![Screen Shot 2020-04-20 at 10 27 48 AM](https://user-images.githubusercontent.com/2441069/79781003-ada19b00-82f1-11ea-8326-fd0b663e4f80.png)

**Comment by DavidBruant** [comment]
This is a Firefox-only issue, hence the difference
Exploration at https://github.com/Agoric/dapp-encouragement/issues/29

**Comment by katelynsills** [comment]
Closed by #30
